manager/manager/utils.py

```diff
@@ -720,3 +781,243 @@ def assert_time_data(time_data):
+    subject : `str`
+        The subject of the email
+    html_content : `str`
+        The content of the email in HTML format
+    plain_content : `str`
+        The content of the email in plain text format
+
+    Notes
+    -----
+    The following environment variables are required to send the email:
+    - SMTP_USER: The SMTP user name
+    - SMTP_PASSWORD: The SMTP user password
+
+    Returns
+    -------
+    bool
+        True if the email was sent successfully, False if not
+    """
+
+    try:
+        # Create message container - the correct MIME type
+        # is multipart/alternative.
+        msg = MIMEMultipart("alternative")
+        part1 = MIMEText(plain_content, "plain")
+        part2 = MIMEText(html_content, "html")
+
+        # Attach parts into message container.
+        # According to RFC 2046, the last part of
+        # a multipart message, in this case
+        # the HTML message, is best and preferred.
+        msg.attach(part1)
+        msg.attach(part2)
+
+        msg["Subject"] = subject
+        msg["From"] = f"{os.environ.get('SMTP_USER')}@lsst.org"
+        msg["To"] = to
+
+        s = smtplib.SMTP("exch-ls.lsst.org", "587")
+        s.starttls()
+        s.login(os.environ.get("SMTP_USER"), os.environ.get("SMTP_PASSWORD"))
+        s.sendmail(msg["From"], msg["To"], msg.as_string())
+        s.quit()
+        return True
+    except Exception:
+        return False
+
+
+def arrange_nightreport_email(report, plain=False):
+    """Arrange the night report email in HTML format
+    or plain text format if specified.
+
+    Parameters
+    ----------
+    report : `dict`
+        The night report data
+
+    Notes:
+    ------
+    The expected parameters of the report dictionary are the following:
+    - telescope: The telescope used during the observing night.
+    Either "AuxTel" or "Simonyi"
+    - day_obs: The observing day in the format "YYYYMMDD"
+    - telescope: The telescope used during the observing night.
+    Either "AuxTel" or "Simonyi"
+    - summary: The summary of the observing night
+    - telescope_status: The final telescope status at the end of the night
+    - confluence_url: The URL of the confluence page with the full report
+    - obs_issues: The list of OBS issues during the night
+    - observers_crew: The list of observers that participated during the night
+
+    Returns
+    -------
+    str
+        The night report email in HTML format or plain text format
+    """
+
+    url_jira_obs_tickets = (
+        "https://rubinobs.atlassian.net/jira/software/c/projects/OBS/boards/232"
+    )
+    day_added = get_obsday_iso(report["day_obs"])
+
+    # TODO: Swap this hardcoded url by a dynamic one.
+    # The service is meant to be run in the summit,
+    # so this will work for the moment.
+    # See: DM-43637
+    url_rolex = f"https://summit-lsp.lsst.codes/rolex?log_date={day_added}"
+
+    if plain:
+        plain_content = f"""
+        Hello everyone!
```

Comment:
Very nice content for the email. However, there are a lot of strings that are repeated for the plain text and HTML templates. Please extract those in constants and use those constants instead. That way, if at some moment the contents of the emails need to be updated, it only needs to be done in one place. You probably want to open a new ticket for that and add a TODO here with a reference to the ticket.